container/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "storage_account_name" {
+  type        = string
+  description = "Storage Account Name"
+}
+
+variable "container_name" {
+  type        = string
+  description = "Storage Container Name"
+}
+variable "container_access" {
+  type        = string
+  description = "Storage Container Access Type"
+  default     = "private"
```

Comment:
i don't think this default is ever used because you've created a type definition for it with it required?

Reply:
variable removed, so not required